README.adoc

```diff
@@ -91,6 +92,66 @@ When a rule matches, the Authorizer will return that rule’s evaluation as a de
 When no rules match, then by default it is denied.
 So, if ACL is not defined for a resource type then it automatically gets denied.
 
+== ManagedKafkaCreateTopicPolicy
+
+The objective of this policy is to:
+
+* validate the topic replication factor (default allows 3 only)
+* validate number of partitions in the cluster
+* validate the create topic configs
+
+Here are the configuration description:
+
+[cols="2,1,5"]
+|===
+|property |default value |description
```

Comment:
Personally I'd have 
```suggestion
|property |description |default value
```

Reply:
```suggestion
|property |description |default value  | default interpretation 
```
re Reading the descriptions. They all describe what the default is, duplicating the column, and generally what that means. for example from `kas.policy.create-topic.max.partitions`: `If this property is not specified, the default of ``-1`` implies no limit.`. It might be clearer to express that using 4 columns. 

```
|kas.policy.create-topic.max.partitions | Specifies the upper limit of partitions that should be allowed in the cluster.| -1 | If this property is not specified, the default of `-1` implies no limit.
```